examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java

```diff
@@ -64,7 +64,7 @@ public KubernetesSeedProvider(Map<String, String> params) {
 
         String host = protocol + "://" + hostName + ":" + hostPort;
         String serviceName = getEnvOrDefault("CASSANDRA_SERVICE", "cassandra");
-        String path = "/api/v1beta1/endpoints/";
+        String path = "/api/v1beta3/endpoints/";
```

Comment:
This is buggy.  I ran across it looking at something else.  You can't just change the API version and not the calling code.  Look 7 lines down - it is trying to decode this object into a v1beta1 structure.

Please audit this and other v1beta3 conversions for this same problem.